packages/zwave-js/src/lib/node/Node.ts

```diff
@@ -2032,6 +2032,44 @@ protocol version:      ${this.protocolVersion}`;
 		return true;
 	}
 
+	/**
+	 * Interviews a command class unless the device configuration provides compat
+	 * values to use instead of performing the interview.
+	 */
+	private async compatInterviewCC(cc: CommandClass): Promise<void> {
+		const compatValues = this._deviceConfig?.compat?.interviewCCs
+			?.get(cc.ccId)
+			?.endpoints.get(cc.endpointIndex);
+		if (!compatValues) {
+			await cc.interview();
+		} else {
+			for (const compatValue of compatValues) {
+				if (cc.isInternalValue(compatValue.property as any)) {
+					const valueId: ValueID = {
+						commandClass: cc.ccId,
+						endpoint: cc.endpointIndex,
+						property: compatValue.property,
+						propertyKey: compatValue.propertyKey,
+					};
+					this.valueDB.setValue(valueId, compatValue.value);
+				} else {
+					this.driver.controllerLog.logNode(
+						this.id,
+						`failed to set compatibility value from device configuration for ${getCCName(
+							cc.ccId,
+						)}, endpoint ${cc.endpointIndex}: '${
+							compatValue.property
+						}' is not a registered internal value`,
+						"error",
+					);
+				}
```

Comment:
I was thinking there needs to be a way to validate the overridden values, but you thought about that already. I also like limiting it to internal values. 👍️